README.md

```diff
@@ -70,52 +99,37 @@ Flags:
 
 Use "terrascan [command] --help" for more information about a command.
 ```
+**Note**: Terrascan will exit with an error code 3 if any issues are found during a scan.
 
 ### Step 3: Integrate with CI\CD
+Terrascan can be integrated into CI/CD pipelines to enforce security best practices.
 Please refer to our [documentation to integrate with your pipeline](https://docs.accurics.com/projects/accurics-terrascan/en/latest/integrations/cicd/).
 
 
 ## Rule Suppression
-If a resource should not be tested against a particular rule, you can tell terrascan to skip it.
+You can configure Terrascan to skip a particular rule while scanning a resource. Follow these steps depending on your platform:
 
 ### Terraform
-In Terraform scripts, you can tell terrascan to skip rules by inserting a comment with the phrase "ts:skip=RULENAME SKIP_REASON". The comment should be inside the resource.
+Use Terraform scripts to configure Terrascan to skip rules by inserting a comment with the phrase `"ts:skip=<RULENAME><SKIP_REASON>"`. The comment should be included inside the resource as shown in the example below.
 
 ![tf](https://user-images.githubusercontent.com/74685902/105115888-847b8a00-5a7e-11eb-983e-7f49f7c36ae1.png)
 
 ### Kubernetes
-In Kubernetes yamls, you can tell terrascan to skip rules by adding an annotation as seen in the snippet below.
+In Kubernetes yamls, you can configure Terrascan to skip rules by adding an annotation as seen in the snippet below.
 
 ![k8s](https://user-images.githubusercontent.com/74685902/105115885-834a5d00-5a7e-11eb-9190-e8b64d77c5ac.png)
 
-### Broad Rule Suppression
-Use our config file to manually pick which rules should be applied or suppressed from the entire scan. This is suitable for edge use cases. Please use in-file suppression to specify resources that shouldn't be tested against particular rules. This ensures that the rules are skipped only for particular resources, rather than all of the resources.
+### Global Rule Suppression
+Use the Terrascan config file to manually select the rules should be excluded or suppressed from the entire scan. This is suitable for edge use cases.
```

Comment:
i think we need to change "excluded or suppressed" to "**included** or suppressed"